recipes/openpmd-viewer/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "openpmd-viewer" %}
+{% set version = "0.9.1" %}
+{% set build = 0 %}
+{% set sha256 = "ac3b2fd00b82fc4ecb2ac7ae991d09fdef3068b0c5e54a08d832c2b7ed65fe6b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/openPMD/openPMD-viewer/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - cython
+    - numpy
+    - setuptools
+  host:
+    - python
+    - pip
+  run:
+    - h5py
+    - matplotlib
+    - {{ pin_compatible('numpy') }}
+    - python
+    - scipy
+    # slider
+    - ipywidgets
+    # openPMD_notebook command
+    - jupyter
+    # examples
+    - wget  # [not win]
+
+test:
+  imports:
+    - opmd_viewer
+  commands:
+    - python -c "import opmd_viewer; print(opmd_viewer.__version__)"
```

Comment:
This is nearly the same as the import test above, can we omit this?

Reply:
That's true, no real added value here besides checking and prining the in-source version again (which I always like, too ^^).